Sofascore2025/Sofascore2025/EventView.swift

```diff
@@ -0,0 +1,263 @@
+        label.textAlignment = .right
+        return label
+    }()
+    
+    private let numAwayTeamGoalsLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 14)
+        label.textAlignment = .right
+        return label
+    }()
+    
+    private let mainView : UIView = {
+        let label = UILabel()
+        label.textAlignment = .center
+        return label
+    }()
+
+    private let mainStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = 0
+        stackView.alignment = .fill
+        return stackView
+    }()
+
+    
+
+
+
+    internal override func addViews() {
+        addSubview(mainStackView)
+        mainStackView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+        
+        mainView.addSubview(homeTeamImageView)
+        mainView.addSubview(leftLabel)
+        mainView.addSubview(awayTeamImageView)
+        //treban li dodati apsolutnom superview-u ili prvom iducem superview-u u hijerarhiji
+        mainView.addSubview(timeLabel)
+        mainView.addSubview(minuteLabel)
+        mainView.addSubview(homeTeamLabel)
+        mainView.addSubview(awayTeamLabel)
+        mainView.addSubview(numHomeTeamGoalsLabel)
+        mainView.addSubview(numAwayTeamGoalsLabel)
+        
+        mainStackView.addArrangedSubview(mainView)
+    }
+
+    
+    internal override func styleViews() {
+        
+        timeLabel.snp.makeConstraints { make in
+            make.centerX.equalTo(leftLabel)
+            make.width.equalTo(56)
+            make.height.equalTo(16)
+            make.top.equalTo(leftLabel).offset(10)
+        }
+        
+        minuteLabel.snp.makeConstraints { make in
+            make.centerX.equalTo(leftLabel)
+            make.top.equalTo(timeLabel.snp.bottom).offset(4)
+            make.width.equalTo(56)
+            make.height.equalTo(16)
+        }
+        
+        leftLabel.snp.makeConstraints { make in
+            make.leading.equalTo(mainView)
+            make.width.equalTo(64)
+            make.height.lessThanOrEqualTo(mainView)
+        }
+        
+        homeTeamLabel.snp.makeConstraints { make in
+            make.width.equalTo(192)
+            make.height.equalTo(16)
+            make.leading.equalTo(leftLabel.snp.trailing).offset(40)
+            make.top.equalTo(mainView).offset(10)
+        }
+        
+        awayTeamLabel.snp.makeConstraints { make in
+            make.width.equalTo(192)
+            make.height.equalTo(16)
+            make.leading.equalTo(leftLabel.snp.trailing).offset(40)
+            make.top.equalTo(homeTeamLabel.snp.bottom).offset(4)
+        }
+        
+        homeTeamImageView.snp.makeConstraints { make in
+            make.top.equalTo(mainView).offset(10)
+            make.leading.equalTo(leftLabel.snp.trailing).offset(16)
+            make.width.height.equalTo(16)
```

Comment:
mozes ovo skratit ovak 
```suggestion
            make.size.equalTo(16)
```